commcare_connect/opportunity/visit_import.py

```diff
@@ -208,17 +208,48 @@ def get_status_by_visit_id(dataset) -> dict[int, VisitValidationStatus]:
     return status_by_visit_id, reason_by_visit_id
 
 
+def get_data_by_visit_id(dataset) -> dict[int, any]:
```

Comment:
nit: there are enough keys and they are strict enough here that it might make sense to return a dataclass or named tuple for stronger typing

Reply:
Added a dataclass to do this. [737592c](https://github.com/dimagi/commcare-connect/pull/383/commits/737592c1941610838a0c164924504d85b7511dcc)